samples/kotlin-android/build.gradle.kts

```diff
@@ -12,6 +12,12 @@ android {
         versionCode = 1
         versionName = "1.0"
     }
+    buildTypes {
+        getByName("release") {
+            isMinifyEnabled = true
+            proguardFiles("moshi.pro")
+        }
+    }
```

Comment:
Why this? I don't think this is needed here (or at least I don't understand why you add it here and not to `groovy-android` or `kotlin-coroutines` as well).

Reply:
The config are not present on groovy-android because I forgot, while kotlin-coroutines did not made me thing to an android "project".

Still I think that is going to be simpler if we just provide one example of proguard rules.
As ideally the way forward will be only moshi-codegen I'm actually going to remove the definitions from kotlin-android